pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
```

Comment:
maybe can we avoid this file and use package.json configuration file instead?

Reply:
This file is needed for pnpm to do the installation and figure out that it is a monorepo, so I am not sure if it can be avoided